### PR TITLE
feat: sidebar toggle mode

### PR DESCRIPTION
**Pull request by rebelchris**
This is the related API change for the sidebar toggle mode.
Related to frontend changes: https://github.com/dailydotdev/apps/pull/721

### CODE REVIEW

File: src/schema/settings.ts [hunk 1]
```diff
@@ -123,6 +130,11 @@ export const typeDefs = gql`
     Whether to open articles on new tab
     """
     openNewTab: Boolean
+
+    """
+    Whether to show the sidebar open
+    """
+    openSidebar: Boolean!
```
Comment (sshanzel) [inline src/schema/settings.ts:137]: This shouldn't be required just like the others.

```suggestion
    openSidebar: Boolean
```

**Review by sshanzel — CHANGES_REQUESTED** [review]
Just this one thing and update the snapshots then this should be good to go 🚀

**Review by idoshamun — APPROVED** [review]
Looking good!
I love the fact that we don't have to change anything on the query once we introduce a new column.

**Review by sshanzel — APPROVED** [review]
LGTM! 🚢

**Pull request merged**